File: Examples/test-suite/ruby/template_extend2_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# Author::    gga
# Copyright:: 2007
# License::   SWIG
#

require 'swig_assert'

require 'template_extend2'

a = Template_extend2::LBaz.new
b = Template_extend2::DBaz.new

raise RuntimeError unless a.foo() == "lBaz::foo"
raise RuntimeError unless b.foo() == "dBaz::foo"

File: Examples/test-suite/javascript/template_extend2_runme.js
var template_extend2 = require("template_extend2");

a = new template_extend2.lBaz();
b = new template_extend2.dBaz();

if (a.foo() != "lBaz::foo") {
    throw new Error;
}

if (b.foo() != "dBaz::foo") {
    throw new Error;
}
